packages/BetterPhpDocParser/ValueObject/Parser/BetterTokenIterator.php

```diff
@@ -102,14 +102,6 @@ public function currentPosition(): int
         return $this->privatesAccessor->getPrivateProperty($this, self::INDEX);
```

Comment:
we can also use `$this->currentTokenIndex()` here

Reply:
it seems not, as that try to call private property from parent class, but that can be on separate PR for re-check :+1:

Reply:
lets see https://github.com/rectorphp/rector-src/pull/3656